core/templates/dev/head/pages/exploration_player/exploration_player.html

```diff
@@ -16,6 +16,10 @@
         '{{exploration_version|js_string}}');
       GLOBALS.INTERACTION_SPECS = JSON.parse(
         '{{INTERACTION_SPECS|js_string}}');
+      GLOBALS.canEdit = JSON.parse(
+        '{{can_edit|js_string}}');
```

Comment:
80 characters in a line. I guess we can shift line 20 to 19. isn't it? (see line 12 for help)

Reply:
Yeah sure we can, I was just following previous fashion. as in GLOBALS.INTERACTION_SPECS